packages/suite/src/components/wallet/CoinmarketLayout/index.tsx

```diff
@@ -18,8 +20,9 @@ interface Props {
 
 const CoinmarketLayout = ({ children }: Props) => {
     const selectedAccount = useSelector(state => state.wallet.selectedAccount);
+    const intl = useIntl();
     return (
-        <WalletLayout title="Coinmarket" account={selectedAccount}>
+        <WalletLayout title={intl.formatMessage(messages.TR_NAV_TRADE)} account={selectedAccount}>
```

Comment:
it would be cleaner to enforce `Translation[id]` as a "title" prop inside of `WalletLayout` and use it as:
`<WalletLayout title="TR_NAV_TRADE" />`

this applies to all files below